src/Loader.php

```diff
@@ -325,9 +325,9 @@ protected function beginsWithAQuote($value)
     public function getEnvironmentVariable($name)
     {
         switch (true) {
-            case array_key_exists($name, $_ENV):
+            case isset($_ENV[$name]):
```

Comment:
What's not correct? It works correctly and fixes two test cases for me over the current code.

I ran my test case through 3v4l to make sure it worked on all versions: https://3v4l.org/ZP3gR

> Error for orig method - expected: 'both_value_server_not_null'; found: NULL
> Error for orig method - expected: 'all_value_both_null_getenv'; found: NULL